tools/fuchsia/copy_debug_symbols.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+""" Gather the build_id, prefix_dir, and exec_name given the path to executable
+    also copies to the specified destination.
```

Comment:
Can you also add a line about how the build ID directory is structure. That section from the presentation will work great :)

Reply:
Done